src/tests/observers/slack_resource_test.cpp

```diff
@@ -56,7 +56,29 @@ TEST(SlackResourceObserver, BasicTest) {
       EXPECT_CALL(mockSink, consume(_)).WillOnce(BasicTestAction(param));
     }
   }
-  jsonSource.RunTests("tests/fixtures/slack_calculation_test.json");
+  jsonSource.RunTests("tests/fixtures/slack_estimator/"
+                          "slack_calculation_test.json");
```

Comment:
I'm not sure, if it won't confuse reader.
I think, this could give an impression of two parameters passed to function. With explicit tab I emphasize that there is a continuation.